### DISCUSSION TITLE
Is it possible remove or disable hyperlinks in PDF

### DISCUSSION THREAD

**Discussion by ademyalcin27** [mozilla/pdf.js#17798] [Q&A]
I want to prevent click links. Is it possible to remove or disable hyperlinks in PDF?

Attach (recommended) or Link to PDF file here:  

Configuration:
- Web browser and its version:
- Operating system and its version:
- PDF.js version: 3.11.174
- Is a browser extension:

Steps to reproduce the problem:
1. 
2.

What is the expected behavior? (add screenshot)

What went wrong? (add screenshot)
![Screenshot 2024-03-15 at 22 01 44](https://github.com/mozilla/pdf.js/assets/5026172/b777a871-551a-4ec0-8e28-a2557e19cda9)

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by nicolo-ribaudo** [comment]
You can set `pointer-events: none` through CSS to all `a` tags inside the viewer.

**Reply by ademyalcin27** [reply]
@nicolo-ribaudo  Yes, I know it but there might be a good solution for security reasons.

**Comment by lasseborly** [comment]
If pointer-events: none is not good enough I would just iterate through all a tags in DOM and replace them.
